[2021-07-18 05:05:25,170] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 05:05:25,590] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 05:05:25,590] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 05:05:25,590] {taskinstance.py:1088} INFO - Starting attempt 4 of 4
[2021-07-18 05:05:25,590] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 05:05:25,722] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_4> on 2021-07-17T03:39:23.918881+00:00
[2021-07-18 05:05:25,726] {standard_task_runner.py:52} INFO - Started process 1854 to run task
[2021-07-18 05:05:25,736] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-07-17T03:39:23.918881+00:00', '--job-id', '49', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp37t7r5c_', '--error-file', '/tmp/tmpwvwdwbj4']
[2021-07-18 05:05:25,737] {standard_task_runner.py:77} INFO - Job 49: Subtask task_4
[2021-07-18 05:05:26,939] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-07-17T03:39:23.918881+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-18 05:05:28,451] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T03:39:23.918881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T03:39:23.918881+00:00
[2021-07-18 05:05:28,451] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-18 05:05:28,452] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-18 05:05:28,459] {subprocess.py:75} INFO - Output:
[2021-07-18 05:05:31,461] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-18 05:05:32,312] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210717T033923, start_date=20210718T050525, end_date=20210718T050532
[2021-07-18 05:05:33,014] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-18 05:05:33,028] {local_task_job.py:151} INFO - Task exited with return code 0
